app/controllers/api/consents_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Api::ConsentsController < ApplicationController
+  skip_before_action :verify_authenticity_token, raise: false
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+
+  def create
+    member = Member.find_by_id_and_email! params[:id], params[:email]
+    member.update(consented_at: Time.now)
```

Comment:
If you're not checking the result of the update is a good practice to use the bang method `update!`